tests/api/v1/botData/get.js

```diff
@@ -0,0 +1,236 @@
+
+  beforeEach((done) => {
+    RoomType.create(rt.getStandard())
+    .then((roomType) => {
+      saveRoomType = roomType;
+      const room = r.getStandard();
+      room.type = roomType.id;
+      return Room.create(room);
+    })
+    .then((room) => {
+      testBotData.roomId = room.id;
+      return Bot.create(b.getStandard());
+    })
+    .then((bot) => {
+      testBotData.botId = bot.id;
+      return BotData.create(testBotData);
+    })
+    .then((botData) => {
+      saveBotData = botData;
+      done();
+    })
+    .catch(done);
+  });
+
+  afterEach(u.forceDelete);
+  after(tu.forceDeleteUser);
+
+  describe('GET bot', () => {
+    it('Pass, get array of one', (done) => {
+      api.get(`${path}`)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.OK)
+      .end((err, res) => {
+        if (err) {
+          done(err);
+        }
+
+        expect(res.body.length).to.equal(ONE);
+        done(err);
+      });
+    });
+
+    it('Pass, get array of multiple', (done) => {
+      const testBotData2 = u.getStandard();
+      testBotData2.name = 'TestData2';
+      testBotData2.botId = testBotData.botId;
+      testBotData2.roomId = testBotData.roomId;
+      BotData.create(testBotData2)
+      .then(() => {
+        api.get(`${path}`)
+        .set('Authorization', token)
+        .expect(constants.httpStatus.OK)
+        .end((err, res) => {
+          if (err) {
+            done(err);
+          }
+
+          expect(res.body.length).to.equal(TWO);
+          done();
+        });
+      })
+      .catch(done);
+    });
+
+    it('Pass, get by name', (done) => {
+      const testBotData2 = u.getStandard();
+      testBotData2.name = 'TestData2';
+      testBotData2.botId = testBotData.botId;
+      testBotData2.roomId = testBotData.roomId;
+      BotData.create(testBotData2)
+      .then(() => {
+        api.get(`${path}?name=`+u.name)
+        .set('Authorization', token)
+        .expect(constants.httpStatus.OK)
+        .end((err, res) => {
+          if (err) {
+            done(err);
+          }
+
+          expect(res.body.length).to.equal(ONE);
+          expect(res.body[ZERO].name).to.equal(u.name);
+          done();
+        });
+      })
+      .catch(done);
+    });
+
+    it('Pass, get data by room', (done) => {
+      const testBotData2 = u.getStandard();
+      const room = r.getStandard();
+      room.name = 'NewRoom';
+      room.type = saveRoomType.id;
+      Room.create(room)
+      .then((newRoom) => {
+        testBotData2.name = 'TestData2';
+        testBotData2.botId = testBotData.botId;
+        testBotData2.roomId = newRoom.id;
+        return BotData.create(testBotData2);
+      })
+      .then(() => {
+        api.get(`/v1/rooms/${testBotData.roomId}/data`)
+        .set('Authorization', token)
+        .expect(constants.httpStatus.OK)
+        .end((err, res) => {
+          if (err) {
+            done(err);
+          }
+
+          expect(res.body.length).to.equal(ONE);
```

Comment:
Do you intend to check the contents here as well? Or that implementation is still pending?